api/organisations/models.py

```diff
@@ -7,12 +7,14 @@
 from django.db import models
 from django.utils import timezone
 from django.utils.encoding import python_2_unicode_compatible
+from django_lifecycle import AFTER_CREATE, AFTER_SAVE, LifecycleModel, hook
 
 from organisations.chargebee import (
     get_customer_id_from_subscription_id,
     get_max_seats_for_plan,
     get_portal_url,
 )
+from users.utils.mailer_lite import mailer_lite
```

Comment:
Seems like we have a mix of different uses of this class. Here we're using it by importing the instantiated client from the util module but elsewhere we're instantiating it directly in the consuming module. Can we be consistent with this? (My preference would be to instantiate it in each consuming module unless there is a compelling argument not to).